cmd/stack_ls.go

```diff
@@ -88,12 +115,42 @@ func newStackLsCmd() *cobra.Command {
 	}
 	cmd.PersistentFlags().BoolVarP(
 		&jsonOut, "json", "j", false, "Emit output as JSON")
+
 	cmd.PersistentFlags().BoolVarP(
 		&allStacks, "all", "a", false, "List all stacks instead of just stacks for the current project")
 
+	cmd.PersistentFlags().StringVarP(
+		&orgFilter, "organization", "o", "", "Filter returned stacks to those in a specific organization")
+	cmd.PersistentFlags().StringVarP(
+		&projFilter, "project", "p", "", "Filter returned stacks to those with a specific project name")
+	cmd.PersistentFlags().StringVarP(
+		&tagFilter, "tag", "t", "", "Filter returned stacks to those in a specific tag (tag-name or tag-name:tag-value)")
+
 	return cmd
 }
 
+// parseTagFilter parses a tag filter into its separate name and value parts, assuming they are separated by a colon.
+// This requires that the tag name not contain a colon, but the tag's value can. Returns an error if the tag filter is
```

Comment:
Don't a majority of the tags we add automatically today contain colons?  Could we use equal or something as a delimiter instead?

Reply:
The equal sign is a _much_ better idea...